spec/features/coworking_feature_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+
+feature 'Co-working' do
```

Comment:
I would have expected the 'feature' test to be more 'feature-y' e.g. load the page, allocate a co-worker, look at the allocations page to see that it shows up.

Reply:
@starswan I don't think that I can allocate a co-worker at the moment as their needs to be changes to the create_or_update method in AllocationService, currently it only adds/updates on primary_pom

Reply:
Ok understand